docs/src/pages/components/progress/CustomizedProgressBars.js

```diff
@@ -1,49 +1,38 @@
 import * as React from 'react';
-import { makeStyles, withStyles } from '@material-ui/core/styles';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import LinearProgress from '@material-ui/core/LinearProgress';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import CircularProgress, {
+  circularProgressClasses,
+} from '@material-ui/core/CircularProgress';
+import LinearProgress, {
+  linearProgressClasses,
+} from '@material-ui/core/LinearProgress';
 
-const BorderLinearProgress = withStyles((theme) => ({
-  root: {
+const BorderLinearProgress = styled(LinearProgress)(
+  ({ theme, styleProps = {} }) => ({
```

Comment:
I am confused with `styleProps`, what are all the accessories that `style` provide?